services/user.py

```diff
@@ -0,0 +1,51 @@
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None) -> User:
+    user = User.objects.create_user(
+        username,
+        email=email,
+        password=password)
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()
+    return user
+
+
+def get_user(user_id: int) -> User | None:
+    try:
+        return User.objects.get(id=user_id)
+    except User.DoesNotExist:
+        return None
```

Comment:
Use the `User.objects.filter` method instead of `User.objects.get` because you're looking for a user by ID, and a user with that ID might not exist. This will avoid you throwing exceptions.

```suggestion
     return User.objects.filter(id=user_id).first()
```